.rubocop_todo.yml

```diff
@@ -24,7 +24,7 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 11
 
-# Offense count: 154
+# Offense count: 151
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Metrics/LineLength:
```

Comment:
Let's set this to 100, which is consistent across all Percy codebases. Offending lines should probably be fixed.